src/emucore/Switches.cxx

```diff
@@ -16,6 +16,8 @@
 // $Id: Switches.cxx,v 1.7 2007/01/01 18:04:50 stephena Exp $
 //============================================================================
 
+#include <cstdint>
```

Comment:
I believe `Switches.hxx` also uses `uint8_t`. Could you move this import into `Switches.hxx` instead?

Reply:
Sure, you're correct, done.